.travis.yml

```diff
@@ -1,64 +1,48 @@
 language: cpp
 
+cache: ccache
+
+
 matrix:
   fast_finish: true
   include:
   - os: linux
     dist: trusty
+    group: stable
     env: BUILDTYPE=Debug
-    addons:
-      apt:
-        packages:
-        - bc
-        - cmake  
-        - libprotobuf-dev
-        - protobuf-compiler
-        - qt5-default
-        - qttools5-dev
-        - qttools5-dev-tools
-        - qtmultimedia5-dev
-        - libqt5multimedia5-plugins
-        - libqt5svg5-dev
-        - libqt5sql5-mysql
   - os: linux
     dist: trusty
+    group: stable
     env: BUILDTYPE=Release
-    addons:
-      apt:
-        packages:
-        - bc
-        - cmake  
-        - libprotobuf-dev
-        - protobuf-compiler
-        - qt5-default
-        - qttools5-dev
-        - qttools5-dev-tools
-        - qtmultimedia5-dev
-        - libqt5multimedia5-plugins
-        - libqt5svg5-dev
-        - libqt5sql5-mysql
+    if: branch = master
   - os: osx
-    osx_image: xcode7.3
+    osx_image: xcode8
```

Comment:
Updating to xcode 8 means cockatrice will only run on osx => 10.12 sierra

Reply:
As said above:
>Update to **last xcode image with 10.11 support** (Xcode 8 GM):
>https://docs.travis-ci.com/user/reference/osx/#OS-X-Version

from the [build log](https://travis-ci.org/Cockatrice/Cockatrice/jobs/331430456#L4):
`instance: 06841a6e-1629-4a9d-81d1-42a7e6d7b3d6 travis-ci-osx10.11-xcode8-1479248015`

Only starting with `xcode8.3` onwards it'll be osx =>10.12!

<br>

Is it worth to consider switchting to 10.12 yet? It cuts off some older hardware and 10.11 is just ~2 years old I think.

Reply:
Reading Apple's docs it says that xcode 8 only installs the SDK for macOS >=10.12, but probably travis handles this different. Sorry for the noise.